examples/create-setters-simple/README.md

```diff
@@ -2,8 +2,7 @@
 
 ### Overview
 
-The `create-setters` KRM config function adds comments to resource fields
-with setter references.
+The `create-setters` KRM config function adds setter comments to resource fields to be parameterized.
 
 In this example, we will see how to add setter comments declaratively to
 resource fields using `create-setters` function.
```

Comment:
I know this is improving an existing example, but in general, examples are more effective if they are closer to the real user scenario.

One CUJ is:
 - How to parameterize the version (tag) of the container image being used in a deployment resource.

Also, would recommend to use familiar resource such as `Deployment` in the examples. I see this example uses `MyKind` which is a custom resource I think.